recipes/feedstockrot/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "feedstockrot" %}
+{% set version = "1.0.0" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: feedstockrot-{{ version }}.tar.gz
+  url: https://github.com/axiom-data-science/feedstockrot/archive/{{ version }}.tar.gz
+  sha256: 32181231186dce986cfd9684001cb63d1411e9a8ca531e129b3600f875cce6ea
+
+build:
+  skip: true  # [not py35]
```

Comment:
Fair enough. I had to ask :grimacing: 

Playing with it now. Very useful! Thanks again for putting this together!!